### ISSUE TITLE
Demo example doesn't build

### ISSUE THREAD

**Issue by DK99** [google/autocxx#165]
## Expected Behavior
I tried to build the demo example. If I just clone the repository everything works.
I would expect the same behavior when using the remote dependencies instead of the local ones.

## Actual Behavior
The build crashes with the following error message
```
  thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ParseError(MacroParseFail(Parsing(Error("expected identifier"))))', demo/build.rs:20:63
  stack backtrace:
     0: rust_begin_unwind
               at /rustc/7eac88abb2e57e752f3302f02be5f3ce3d7adfb4/library/std/src/panicking.rs:483
     1: core::panicking::panic_fmt
               at /rustc/7eac88abb2e57e752f3302f02be5f3ce3d7adfb4/library/core/src/panicking.rs:85
     2: core::option::expect_none_failed
               at /rustc/7eac88abb2e57e752f3302f02be5f3ce3d7adfb4/library/core/src/option.rs:1234
     3: core::result::Result<T,E>::unwrap
               at /rustc/7eac88abb2e57e752f3302f02be5f3ce3d7adfb4/library/core/src/result.rs:973
     4: build_script_build::main
               at ./build.rs:20
     5: core::ops::function::FnOnce::call_once
               at /rustc/7eac88abb2e57e752f3302f02be5f3ce3d7adfb4/library/core/src/ops/function.rs:227
  note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

## Steps to Reproduce the Problem

1. Clone the repository
1. Edit the `demo/Cargo.toml` file and replace the local dependencies in lines 23 and 26 with remote dependencies
1. Execute `cargo build`

## Specifications

- Version: Rust 1.48.0
- Platform: Arch Linux

**Comment by adetaylor** [comment]
Thanks for the report!

There's been a breaking change between the last release (0.4.1) and the version currently in the `main` branch. Remove the `unsafe` keyword from `demo/src/main.rs` and it will build. Version 0.4.1 does not allow that keyword; the next version (likely 0.5) will allow it and possibly even require it.

I want to do a _little_ bit more thinking about the `unsafe` keyword before making that 0.5 `autocxx` release.

**Comment by adetaylor** [comment]
I wrote up #166 to explain myself here :) (i.e. why there isn't yet a 0.5 release)

**Comment by adetaylor** [comment]
I released 0.5 which should resolve this. However, #168 could still present problems - working on that now.
